subcommands/common.go

```diff
@@ -55,6 +52,10 @@ func Login(cmd *cobra.Command) *client.Api {
 		DieNotNil(fmt.Errorf("Required flag \"factory\" not set"))
 	}
 	creds := client.NewClientCredentials(Config.ClientCredentials)
+	if viper.GetBool("server.insecure_skip_verify") {
+		Config.InsecureSkipVerify = true
+		creds.InsecureSSL = true
+	}
```

Comment:
Can you put this block into the desired place in the [original commit](https://github.com/foundriesio/fioctl/pull/239/commits/f93f30446e9593b928410c69f813973f96551072), to make the diff cleaner?

Reply:
fixed in force-push - you actually uncovered a bug i introduced here.